### DISCUSSION TITLE
It needs to support pinch zoom

### DISCUSSION THREAD

**Discussion by MCorbo7** [AppGround-io/bluetooth-keyboard-and-mouse-support#330] [Ideas]
Like if you were using a touch screen in your computer.
Control + isn't the same as it enlarges content destroying layout.

Pinch zoom example:
![image](https://i.stack.imgur.com/CalE5.gif)

Control + example:
![image](https://i.stack.imgur.com/9XiMD.gif)

**Comment by AlanDroide** [comment]
Would be awesome, is it possible with other apps?
